lib/endpoints/class-wp-json-terms-controller.php

```diff
@@ -16,12 +16,29 @@ public function register_routes() {
 				'callback'            => array( $this, 'get_items' ),
 				'permission_callback' => array( $this, 'get_items_permissions_check' ),
 				'args'                => array(
-					'search'   => array(),
-					'per_page' => array(),
-					'page'     => array(),
-					'order'    => array(),
-					'orderby'  => array(),
-					'post'     => array(),
+					'search'   => array(
+						'sanitize_callback' => 'sanitize_text_field',
+						'default'           => '',
+					),
+					'per_page' => array(
+						'sanitize_callback' => 'absint',
+						'default'           => 10,
+					),
+					'page'     => array(
+						'sanitize_callback' => 'absint',
+						'default'           => 1,
+					),
+					'order'    => array(
+						'sanitize_callback' => 'sanitize_key',
+						'default'           => '',
```

Comment:
@joehoyle Any reason why the default here isn't 'ASC' the default for `get_terms()`?